sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/statusdisplay/SampleCount.groovy

```diff
@@ -0,0 +1,28 @@
+package life.qbic.portal.sampletracking.components.projectoverview.statusdisplay
+
+import groovy.transform.EqualsAndHashCode
+
+/**
+ * <b>Stores the number of failing and passing sample counts for a status</b>
+ *
+ * @since 1.0.0
+ */
+@EqualsAndHashCode
```

Comment:
I think it would be a great addition to add tests for equality, to make sure that the equals implementation does not change in the future.

Reply:
added them +1